Tasks/DeployAzureResourceGroup/AzureUtilityGTE1.0.ps1

```diff
@@ -35,15 +35,17 @@ function Deploy-AzureResourceGroup
           [string]$resourceGroupName,
           [string]$overrideParameters)
 
-    Write-Host "[Azure Resource Manager]Creating resource group deployment with name $resourceGroupName"
+    $deploymentName = [System.IO.Path]::GetFileNameWithoutExtension($csmFile) + '-' + ((Get-Date).ToUniversalTime()).ToString('yyyyMMdd-HHmm')
+
+    Write-Host "[Azure Resource Manager]Creating resource group deployment with name $deploymentName"
```

Comment:
The resource group name is displayed earlier (when it is created). This log is for describing the name of the deployment that is about to be created. Make sense?